app/src/main/java/com/xmartlabs/taskloans/ui/screens/dashboard/tabs/HistoryFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.xmartlabs.taskloans.ui.screens.dashboard.tabs
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.xmartlabs.taskloans.R
+import com.xmartlabs.taskloans.databinding.FragmentHistoryBinding
+import com.xmartlabs.taskloans.ui.common.BaseViewBindingFragment
+
+class HistoryFragment : BaseViewBindingFragment<FragmentHistoryBinding>() {
+
+  override fun inflateViewBinding(): FragmentHistoryBinding =
+      FragmentHistoryBinding.inflate(layoutInflater)
+
+  override fun onCreateView(
```

Comment:
i think that you can delete this since you had a `BaseViewBindingFragment` and you already declared `inflateViewBinding()`